src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const daySum = days * 40;
+
+  if (days >= 3 && days < 7) {
+    return daySum - 20;
+  }
+
+  if (days >= 7) {
```

Comment:
Use a constant for the long-term discount threshold (7 days) and the discount amount to avoid hardcoding.